docs/index.rst

```diff
@@ -49,6 +49,57 @@ frontend, like ``builddir``, to specify the build directory to use/re-use. You
 can find more information about them in the `build options page`_.
 
 
+Editable installs
+-----------------
+
+Editable installs allow you to install the project in such a way where you can
+edit the project source and have those changes be reflected in the installed
+module.
+
+The way it works on ``meson-python`` specifically, is that when you import a
+module of the project, it will be rebuilt. This means that project imports will
+take more time than usual.
+
+You can use pip, or other package manager, to install the project in editable
```

Comment:
Is there another installer / package manager that does editable installs? If not, better to leave it out.